tools/cloudformation-template/README.md

```diff
@@ -83,7 +89,7 @@ As you follow the official docs, you can use the recommended configurations belo
 
```

Comment:
I cannot select the line as it is not a part of this change but...

Line 80 says `` - Under **Maximum concurrent accounts**, choose `Percent` and set field value to `100`.``

It should reference 'Failure tolerance' rather than 'Maximum concurrent accounts':
![image](https://github.com/user-attachments/assets/e3b6db1e-66bd-4ad1-9acb-d16467d5a30e)

Therefore line 80 should read `` -  Under **Failure tolerance**, choose `Percent` and set field value to `100`.``

Reply:
Both Max Concurrent Accounts and Failure tolerance should be set to "Percentage" and "100" for the value

Reply:
Yeah but the doc references Max Concurrent Accounts twice, rather than referencing Max Concurrent Accounts once and Failure tolerance once @bryankaraffa

Reply:
![image](https://github.com/user-attachments/assets/d309d066-e3d8-44b1-a4c3-d4435830627b)
See line 80